### PR TITLE
docs: Update Jira docs with new installation instructions

### PR DESCRIPTION
**Pull request by thomasheartman**
This PR updates the JIRA docs with new installation instructions now that the plugin has been published to the marketplace.

## Help

I don't have access to a Jira server myself, so I can't verify that all the steps are correct.

### CODE REVIEW

**Review by chriswk — APPROVED** [review]
Looks good to me

File: website/docs/integrations/jira-server-plugin-installation.md [hunk 1]
```diff
@@ -37,6 +36,19 @@ You will need a Jira admin user.
 
 The [Jira server plugin is available in the Atlassian marketplace](https://marketplace.atlassian.com/apps/1227377/unleash-for-jira?tab=overview&hosting=datacenter).
 
+You'll need to download the plugin and create a license key.
+
+If you have an Unleash enterprise license you're welcome to reach out to us at sales@getunleash.io for a free plugin license, otherwise you'll need to try the plugin for 30 days free or purchase a license through the marketplace.
+
+Once you've downloaded the plugin artifact, you'll need to click the Manage Apps link in Jira's administration menu.
+
+![Jira Manage Apps - Upload App](/img/jira_upload_app.png)
```
Comment (thomasheartman) [inline website/docs/integrations/jira-server-plugin-installation.md:45]:
```suggestion
![The Jira "manage apps" page. The "upload app" button is visually highlighted.](/img/jira_upload_app.png)
```

File: website/docs/integrations/jira-server-plugin-installation.md [hunk 2]
```diff
@@ -37,6 +36,19 @@ You will need a Jira admin user.
 
 The [Jira server plugin is available in the Atlassian marketplace](https://marketplace.atlassian.com/apps/1227377/unleash-for-jira?tab=overview&hosting=datacenter).
 
+You'll need to download the plugin and create a license key.
+
+If you have an Unleash enterprise license you're welcome to reach out to us at sales@getunleash.io for a free plugin license, otherwise you'll need to try the plugin for 30 days free or purchase a license through the marketplace.
+
+Once you've downloaded the plugin artifact, you'll need to click the Manage Apps link in Jira's administration menu.
```
Comment (thomasheartman) [inline website/docs/integrations/jira-server-plugin-installation.md:43]:
```suggestion
Once you've downloaded the plugin artifact, you'll need to follow the Manage apps link in Jira's administration menu.

On the Manage apps page, use the "upload app" button.
```

File: website/docs/integrations/jira-server-plugin-installation.md [hunk 3]
```diff
@@ -37,6 +36,19 @@ You will need a Jira admin user.
...
+
+![Jira Manage Apps - Upload App](/img/jira_upload_app.png)
+
+This will prompt you to select the plugin file you downloaded in the previous steps for upload.
+
+Once you've selected your file and clicked "Upload". Jira will install the plugin, and you should get a nice status screen saying that it successfully installed.
```
Comment (thomasheartman) [inline website/docs/integrations/jira-server-plugin-installation.md:49]:
```suggestion
Once you've selected your file and started the upload, Jira will install the plugin. If the installation is successful, you'll get a status screen saying that it successfully installed.
```

File: website/docs/integrations/jira-server-plugin-installation.md [hunk 4]
```diff
@@ -37,6 +36,19 @@ You will need a Jira admin user.
...
+
+![Jira Manage Apps - Successful install](/img/jira_server_installed_ready_to_go.png)
```
Comment (thomasheartman) [inline website/docs/integrations/jira-server-plugin-installation.md:51]:
```suggestion
![The successful install screen mentioned in the preceding paragraph.](/img/jira_server_installed_ready_to_go.png)
```

**Review by thomasheartman — COMMENTED** [review]
@sighphyre Thanks so much for adding the steps 😄  I've added a few suggestions to add alt text + some minor rewording.

**Pull request merged**